repos/system_upgrade/el7toel8/models/cupsmigrationmodel.py

```diff
@@ -0,0 +1,17 @@
+from leapp.models import Model, fields
+from leapp.topics import SystemInfoTopic
+
+class Matrix(Model):
```

Comment:
Could we have a more descriptive name for this model? Something like CupsMigrationMatrix? That way we can avoid some naming conflicts in the future. Also, some docstring documentation about the model can be helpful.

Reply:
No longer used.